src/_pytest/config/__init__.py

```diff
@@ -256,6 +256,7 @@ def directory_arg(path: str, optname: str) -> str:
     "reports",
     *(["unraisableexception", "threadexception"] if sys.version_info >= (3, 8) else []),
     "faulthandler",
+    "pythonpath",
```

Comment:
I vaguely recall `faulthandler` should be last on this list. I might be misremembering but couldn't hurt to move it up a bit.

```suggestion
    "reports",
    "pythonpath",
    *(["unraisableexception", "threadexception"] if sys.version_info >= (3, 8) else []),
    "faulthandler",
```